assets/i18n/qqq.json

```diff
@@ -44,6 +44,7 @@
     "I18N_FOOTER_CONTRIBUTE_ALL_CAPS": "Text shown in the footer. - Title of the contribute section. Must be in all capital letters for languages that use Latin, Cyrillic, and/or Greek scripts. This section has links to pages where people can contribute to Oppia.\n{{Identical|Contribute}}",
     "I18N_FOOTER_CREDITS": "Text shown in the footer. - Link to the credits tab in the about page. See I18N_ABOUT_PAGE_TITLE\n{{Identical|Credit}}",
     "I18N_FOOTER_DONATE": "Text shown in the footer. - Link to the donation page.\n{{Identical|Donate}}",
+    "I18N_FOOTER_FOLLOW_US": "Text displayed in the Footer. It's shown above social media icon like facebook and youtube.",
```

Comment:
Will correct that, btw translate wiki correct for other languages. And can i edit online (might increase number of commits per PR), sometimes i don't carry my laptop always so i push changes thru browser.